### PR TITLE
SidePanel: move to react-spring

### PR DESCRIPTION
**Pull request by bpierre**
Also export a new `springs` object.

### CODE REVIEW

File: src/utils/styles/spring.js [hunk 1]
```diff
@@ -1,18 +1,41 @@
...
+  slow: { tension: 150, friction: 18 },
+  normal: { tension: 190, friction: 22 },
+  fast: { tension: 220, friction: 24 },
 }
 
-export const spring = (name: SpringName) => {
-  return SPRINGS[name] || SPRINGS.normal
+// Deprecated, see above
+export const spring = name => {
+  return springs[name] || springs.normal
```
Comment (sohkai) [inline src/utils/styles/spring.js]: Might be useful to have a depreciation strategy, like using `console.warn()` if it's not in production mode.

File: src/utils/styles/spring.js [hunk 2]
```diff
@@ -1,18 +1,41 @@
...
 }
+
+const reactMotionSprings = Object.entries(springs).reduce(
```
Comment (sohkai) [inline src/utils/styles/spring.js]: Not sure if this needs to be here; doesn't seem to be getting used? Should we export it?

Comment (bpierre) [reply src/utils/styles/spring.js]: Ah yes it should be used by `spring()` for react-motion. Thanks!

**Comment by bpierre** [comment]
> Do you think it'd be nice to have a doc somewhere that lists the things we're starting to depreciate? A full blown changelog would be great but it's probably too much for now.

Yes it’s a good idea, we should start having a CHANGELOG.md in this project. I will add with the next release.

**Pull request merged**